app.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require("./controllers/controller");
+app.use(express.json())
```

Comment:
It's best practice to only commit code that is necessary/being used. We don't need to use express.json() until we are dealing with a patch/post request. Something to bear in mind going forwards!